kolibri/plugins/learn/assets/src/views/TopicsContentPage.vue

```diff
@@ -302,6 +312,18 @@
       missingLessonResources() {
         return this.lesson && this.lesson.resources.some(c => !c.contentnode);
       },
+      isRemoteContent() {
+        return Boolean(this.deviceId);
+      },
+      isDownloadedRemoteContent() {
+        return this.isRemoteContent && Boolean(this.downloadRequestMap[this.content.id]);
+      },
+      allowRemoteDownload() {
+        return this.isUserLoggedIn && this.isRemoteContent && !this.isDownloadedRemoteContent;
```

Comment:
@rtibbles I saw that in your [HybridLearningFooter](https://github.com/learningequality/kolibri/pull/10129/files#diff-e985e6e0a76dfd6c79d3937a14c90d2e97d2fa6ff5125cb1d5e860db918cd6b3R154-R158) updates, you also used `admin_imported` flag as one of the conditions to determine whether the download should be allowed. Is it something to be used here too? What would it mean in this context?

Reply:
Yes - `admin_imported` means that it has been imported to the device by the admin, and in the designs it prevents users from marking resources for download in this case, so you can add an extra conditional for this case.